eng/PatchConfig.props

```diff
@@ -36,7 +36,6 @@ Later on, this will be checked using this condition:
       Microsoft.AspNetCore.Mvc.Core;
       Microsoft.AspNetCore.Routing;
       Microsoft.AspNetCore.Server.IIS;
-      Microsoft.AspNetCore.Server.Kestrel.Core;
```

Comment:
Just to confirm - nothing else was added to Kestrel.Core since the 2.2.0 release, right?

Reply:
I double-checked and the only other commits in `src/Servers/Kestrel/Core` since 2.2.0 look like code reorganization and test-only changes. For example, the merge commits from `release/2.1` that affected product code just changed the IVTs in `AssemblyInfo.cs`.

However, it looks like a `release/2.1` change (feb641ddda) that was merged into `release/2.2` after 2.2.0 updated `src/Shared/CertificateGeneration/` significantly. Kestrel Core has a wildcard `<Compile/>` for that directory and uses `CertificateManager`. **Is what feb641ddda merged important for 2.2.2?**

Reply:
Nothing changed in Kestrel.Core proper since 2.2.0. Since we're not going to ship Kestrel.Core as part of 2.2.2, I would think that Kestrel.Core's CertificateManager reference is irrelevant.

Reply:
Yeah, we're okay. That change you're seeing is https://github.com/aspnet/AspNetCore/pull/4801, which moved source code from aspnet/Extensions into this repo.